index.d.ts

```diff
@@ -43,10 +45,17 @@ export interface Options extends FastGlobOptions {
 	@default false
 	*/
 	readonly gitignore?: boolean;
+
+	/**
+	The current working directory in which to search.
+
+	@default process.cwd()
+	*/
+	readonly cwd?: URL | string;
 }
 
 export interface GitignoreOptions {
-	readonly cwd?: string;
+	readonly cwd?: URL | string;
```

Comment:
`URL` interface belongs to `lib.dom.d.ts`, so now we force TS users to inject `compilerOptions.lib: ['dom']` to resolve it. It breaks TS builds and formally this is a breaking change.

Reply:
https://nodejs.org/api/url.html#the-whatwg-url-api

Reply:
So it should be imported in `index.d.ts`. It does not belong to global by default.

Reply:
@sindresorhus 
I really don't want to fix hundreds of our `globby@^12`-dependant projects all night )). Could you plz roll the fix #206?

Reply:
I'm sorry for breaking your projects, but I know nothing about the typing, I can't help.